### DISCUSSION TITLE
Slider - do not change value during debounce/delay

### DISCUSSION THREAD

**Discussion by jan-drozen** [microsoft/fluentui-blazor#2492] [Q&A]
Hello everyone,
with this simple setup:

`<FluentSlider Min="0" Max="100" Step="1" @bind-Value=SliderValue @bind-Value:after="WriteValue" Immediate="false" ImmediateDelay="2000"></FluentSlider>`

I would expect that during the delay period (2 sec in this case) the bound value `SliderValue` is not changed and also the `:after` event is also not triggered.

What I would like to avoid is to trigger the `WriteValue` event for each value between the original and the new target value if user drags the slider. Now I can see in the console that the event is triggered immediately and for each step.

I would love to trigger it after the delay period elapsed from the last user change to the control.

![image](https://github.com/user-attachments/assets/2c09115b-110c-4e10-ac34-332b063cc02b)

**Comment by vnbaaij** [comment]
As you have found with the logging result, the `FluentSlider` component does not do anything with the `Immediate` and `ImmediateDelay` parameters.
It is only implemented for the text field based components.
